frontend/src/components/ResultsPage/ResultsPage.js

```diff
@@ -188,7 +190,20 @@ export const ResultsPage = ({
               </div>
             </div>
             <br />
-            <Divider orientation="left">Risk Measure</Divider>
+            <Divider orientation="left">
+              Risk Measure 
+              <Popover content={
+                  <div style={{maxWidth:'28em'}}>
+                    <p><strong>Model:</strong> RandomForestClassifier</p>
+                    <p><strong>AUC ROC</strong>: 91.3%</p>
+                    <p>Area under the ROC curve is interpreted as the probability that a day-grid with fire will receive a higher risk score than a day-grid with no fire.</p>
```

Comment:
I think we need something I literal more understandable added. this is this the accuracy?

Reply:
AUC is often used with highly imbalanced datasets. Accuracy is like 99.998% because the test set is mostly no fire examples. Recall and precision could also be used but they are much worse and require more explanation too

Reply:
lol 99% is impressive. Ok!

Reply:
keep it that way it and lets merge

Reply:
haha, okay! Sorry for late PR